src/site/content/en/blog/persistent-storage/index.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Persistent storage
+subhead: Persistent storage can help protect critical data from eviction, and reduce the chance of data loss.
+authors:
+  - petelepage
+description: Persistent storage can help protect critical data from eviction, and reduce the chance of data loss.
+date: 2020-05-07
+updated: 2020-05-07
+tags:
+  - blog
+  - storage
+  - progressive-web-apps
+hero: hero.jpg
+alt: PCB circuit board of electronic device
+---
+
+When faced with storage pressure like low disk space, browsers will
+typically [evict data][eviction], including from the Cache API and IndexedDB,
+from the least recently used origin. This may cause data loss if the app
+hasn't synced data to the server, and reduce the reliability of the app by
+removing resources required for the app to work. Both of which lead to
+negative user experiences.
+
+Thankfully, research by the Chrome team shows that data is very rarely
+cleared automatically by Chrome. It is far more common for users to manually
+clear storage. Thus if a user visits your site regularly, the chances are
+small that your data will be evicted.
+
+To prevent the browser from deleting your data, you can request that your
+entire site's storage be marked persistent. **You should only do this if you
+have critical data that is not backed up to the cloud, for example end-to-end
+encryption keys, that if deleted could result in significant data loss.**
```

Comment:
If it's important enough to bold the entire sentence, it should be pulled into an aside.

Reply:
I tried that, and I don't like the way it reads if I do. I'd like to leave as if it no major objections.

Reply:
I'm not crazy about bolding sentences in middle of a paragraph. Did you try making the whole paragraph shorter and using that as the aside?

>Requesting that all your site’s data be marked as persistent should only be done for critical data (for example, end-to-end encryption keys) that if not backed up to the cloud, could result in significant data loss if not saved. Persistent storage is not deleted by the browser, even if storage running low. It’s only deleted if the user choses to do so.

Reply:
Fine, I still don't love it, but don't want to go back and forth. Moved it to an aside.